src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -179,7 +180,8 @@ trait SharedFileSystem {
 
   private def stageWdlFile(call: Option[Call], wdlFile: WdlFile, hostInputsPath: Path): Try[WdlFile] = {
     val originalPath = Paths.get(wdlFile.value)
-    val executionPath = hostInputsPath.resolve(originalPath.getFileName.toString)
+    val uniqueID = UUID.randomUUID().toString.split("-")(0)
```

Comment:
question, not a suggestion. Would it make sense to have an ID on the instance of the Call and use that, like we do w/ workflow?